GameBoard/Models/GameEventInviteForm/GameEventInviteFormViewModel.cs

```diff
@@ -8,11 +10,23 @@ public class GameEventInviteFormViewModel
 
         public string UserName { get; }
 
-        public GameEventInviteFormViewModel(int gameEventId, string gameEventName, string userName)
+        public int GroupId { get; }
```

Comment:
You already have access to the `Id` via `Group.Id`.

Reply:
Resolved

Reply:
GroupID removed